### DISCUSSION TITLE
Initial Setup requires External domain address

### DISCUSSION THREAD

**Discussion by slacker247** [nextcloud/all-in-one#1155] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1.  Create a docker instance using the steps outlined in the ReadMe.md
2.  Navigate to the nextCloud instance running on docker
3.  Click button saying "Open NextCloud AIO login
4.  login
5.  NextCloud AIO instance requests a domain for Next Cloud
6.  FAIL

### Expected behavior 
I have an internal DNS.  If I put an internal domain nextcloud.local the install complains
If I put my external domain.  the installer complains that it's internal.  why because the DNS points directly to my web server.  
No need to go outside the network to get to the web server that is inside the network.

### Actual behavior 
It wants an external ip returned for the provided DNS name

### Host OS 
docker

#### Nextcloud AIO version 
2.0.2

#### Current channel 
???

#### Other valuable info 
what's the point of forcing it to have an external facing domain name?
I can easily route any dedicated ports to the nextcloud internal server.
I want to route from my web server to the next cloud server
[internet] - [IDS] - [web server] - [internal servers such as next cloud]
web server does a route on url path request such as www.webserver.net/nextcloud -> nextcloud.local

**Comment by szaimen** [comment]
This is not a bug but a feature. It is the only way how we can make sure that the domain is correctly configured. However there are multiple workarounds. One of which is this: https://github.com/nextcloud/all-in-one#how-to-skip-the-domain-validation

**Reply by slacker247** [reply]
Ok, thanks.  I didn't even look for an answer because the UI made it seem like it was the only way forward.
Maybe add a note on the UI saying
* If you are an advanced user, there is a was to bypass this step.
Which would indicate that they need to go look for the answer.

**Reply by szaimen** [reply]
There actually is an advice in the UI how you can skip the domain verification...

**Reply by slacker247** [reply]
I didn't see anything that suggested that.
but it doesn't matter.  I dropped your product for filerun
yours is now unable to start the apache server due to the inability to create an ssl
filerun to me seconds to get setup and working.

**Reply by szaimen** [reply]
I guess it doesnt matter then.

**Reply by szaimen** [reply]
Good luck with filerun!

**Comment by szaimen** [comment, marked as answer]
For anyone that comes by, the OP was prpbably looking for this: https://github.com/nextcloud/all-in-one/discussions/588#discussioncomment-3450343

However supported is only this: https://github.com/nextcloud/all-in-one/blob/main/local-instance.md
